### ISSUE TITLE
Contact Us- NOT RESPONSIVE

### ISSUE THREAD

**Issue by Sneha6003** [sayandippaul/IRIS-Smart-Tutor#1]
The contact us feature is not responsive. Please assign me this issue to work on.
Thank you!

**Comment by sayandippaul** [comment]
Pls answer
I have a question

Did you get any discord channel link joining for jwoc 24

Became I didn't get any link for that ...if you got that link pls share with me .....

And I am looking your issue and trying to assign that

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Sneha6003 ***@***.***>
Sent: Thursday, January 11, 2024 10:45:19 PM
To: sayandippaul/IRIS-Smart-Tutor ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sayandippaul/IRIS-Smart-Tutor] Contact Us- NOT RESPONSIVE (Issue #1)


The contact us feature is not responsive. Please assign me this issue to work on.
Thank you!

**Comment by Sneha6003** [comment]
Yes I received the discord link, Ill contact the admin and will send you
the link.

**Comment by Sneha6003** [comment]
Here is the discord link,

https://discord.gg/VsxcwfgR

Please dm the organizer once you join.
Thank you

**Comment by Sneha6003** [comment]
I would like to work on that issue under JWOC.

**Comment by sayandippaul** [comment]
delete this issue and reopen this issue again 
else this will not countable

**Comment by sayandippaul** [comment]
@Sneha6003
